quickcomm/pagination.py

```diff
@@ -0,0 +1,54 @@
+
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+class BasePagination(PageNumberPagination):
+    page_size_query_param = 'size'
```

Comment:
not sure if I'm just missing something but how does this handle the case of users asking for a specific page?